tox.ini

```diff
@@ -1,15 +1,15 @@
-[flake8]
-max-line-length = 99
-
 [tox]
 envlist = py36, py37, py37-flake8
 
+[flake8]
+max-line-length = 99
+
 [testenv]
-usedevelop=True
-deps = -r{toxinidir}/requirements/requirements.txt
-       -r{toxinidir}/requirements/requirements-test.txt
+deps = -rrequirements-test.txt
+       pip >= 18.0.1
 commands = pytest
 
 [testenv:py37-flake8]
 deps = flake8
+       pip >= 18.0.1
```

Comment:
I don't think we need this here. I believe Python 3.7 comes with a recent-enough version of pip already.